nextcloud/user_saml
# detect disabled user and show a apropriated error message

detect disabled user and show a appropriated error message

Steps to test:

1. setup Nextcloud with SAML
2. Disable a saml user in the user management
3. Try to login with the disabled user

Until now: it result in a redirect loop between Nextcloud and the idp

With this PR: a appropriated error message is shown.
